readthedocs/templates/projects/notifications/deprecated_config_file_used_email.txt

```diff
@@ -0,0 +1,9 @@
+{% extends "core/email/common.txt" %}
+{% block content %}
+Your project "{{ project.slug }}" is not using a <a href="https://docs.readthedocs.io/en/stable/config-file/v2.html">.readthedocs.yaml</a> configuration file and will stop working soon.
```

Comment:
I guess this is also an issue on the versions of this above, but the link text should cover all of this.

```suggestion
Your project "{{ project.slug }}" is not using a <a href="https://docs.readthedocs.io/en/stable/config-file/v2.html">.readthedocs.yaml configuration file</a> and will stop working soon.
```

Reply:
I just realized that it doesn't make sense to have an `<a>` on a .txt file, so I changed this for a simple link without HTML tags 😄